womenwhogo.org/scholarships.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Women Who Go scholarship application">
+    <meta name="keywords" content="Women Who Go, Go, Golang, Women in Technology, WWG, Go, Programming, Events, scholarship">
+
+    <meta name="google-site-verification" content="D_yba5vBXKgsE50Rcl00dhHgrHZttf2zB0-TSAYW8hY" />
+
+    <link rel="icon" href="favicon.ico">
+
+    <title>Scholarhips - Women Who Go</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
+
+    <link href="./assets/css/index.css" rel="stylesheet">
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="//oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="//oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <div class="container header">
+      <div class="row"><p class="text-center col-xs-6 col-xs-offset-3">
+        <a href="/">
+          <img class="main-img" src="assets/images/wwglogo.png">
+        </a>
+      </p></div>
+      <h1 class="text-center">GopherCon Scholarhip</h1>
+    </div>
+    <hr>
+    <br>
+    <br>
+    <div class="container links">
+      <h2>Eligibility</h2>
+        <p>You must meet at least one of the following requirements:</p>
+        <ul>
+          <li>Identify as a woman</li>
+          <li>Identify as a person of color</li>
+          <li>Identify as LGBTQ+ or non-binary</li>
+          <li>Identify as disabled</li>
+        </ul>
+      <h2>What's Covered</h2>
+      <p>Our goal is to send as many people as possible to GopherCon.
+        Each scholarship is different, depending on what you need help with:
+      </p>
+      <ul>
+        <li>GopherCon ticket</li>
+        <li>Plane tickets</li>
+        <li>Hotel/Airbnb</li>
+        <li>Food, taxi, etc.</li>
+      </ul>
+      <p>If your employer can pay for any part of your expenses, we ask that you take advantage of that
+        first and use the scholarship for only uncovered expenses.
+      </p>
+      <p>
+      <div style="margin-bottom: 3em;">
+        <h2 style="text-align: center">
+          <a href="https://docs.google.com/forms/d/1CukfD-XPLJ87pzz1qmKCuoN6mSKSpVgvu6lydCzFNn0">GopherCon US 2018 Application Form</a>
+        </h2>
+      </div>
+    </div>
+
+    <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
i wonder who owns this GA tag....

Reply:
I am guessing Sarah?

Reply:
If so we should be able to log in to it. Will look into out GSuite later to see if it is there